Library/brew.sh

```diff
@@ -1,3 +1,36 @@
+read -d "" HOMEBREW_HELP <<EOS
```

Comment:
I think it'd be good to pull this into `Library/bash/help.sh` and `source` it here.